build.cake

```diff
@@ -26,27 +26,59 @@ Task("Clean")
 Task("Restore")
     .Does(() =>
     {
-        DotNetCoreRestore(solutionPath);
+		 if ( !DirectoryExists(nuget) )
+		 {
+			CreateDirectory(nuget);
+		 }
+
+		NuGetRestore(solutionPath);
     });
 
 // Build using the build configuration specified as an argument.
  Task("Build")
     .Does(() =>
     {
-        DotNetCoreBuild(solutionPath,
-            new DotNetCoreBuildSettings()
-            {
-                Configuration = configuration,
-                ArgumentCustomization = args => args.Append("--no-restore"),
-            });
+		MSBuild(solutionPath, new MSBuildSettings {
```

Comment:
Could you adjust the indentation, please?

Reply:
Hopefully I fixed it: Could not see the difference!